File: wxrtc/src/main/java/com/wx/rtc/bean/RoomMsg.java
package com.wx.rtc.bean;

import com.google.gson.annotations.SerializedName;

public class RoomMsg {
    @SerializedName("cmd")
    private String cmd;
    @SerializedName("message")
    private String message;

    public String getCmd() {
        return cmd;
    }

    public void setCmd(String cmd) {
        this.cmd = cmd;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: wxrtc/src/main/java/com/wx/rtc/bean/RoomMsg.kt
package com.wx.rtc.bean

import com.google.gson.annotations.SerializedName

internal class RoomMsg {
    @JvmField
    @SerializedName("cmd")
    var cmd: String? = null

    @JvmField
    @SerializedName("message")
    var message: String? = null
}
